sql/driver/conn.go

```diff
@@ -112,6 +109,9 @@ func (c *conn) send(args Request) (*rows, error) {
 		return r, nil
 	}
 	result := resp.Results[index]
+	if result.Error != nil {
```

Comment:
I think when you are using the driver you are discouraged from using it with multiple statements. I think we should not do anything fancy for a user who wants to use it in this way. Let's stick with reporting whatever the last result says.